content/en/synthetics/settings/_index.md

```diff
@@ -102,18 +102,32 @@ To generate and use a TOTP in your tests, create a global variable where you ent
 
 1. In **Choose variable type**, select **MFA Token**.
 2. In **Define Variable**, enter a **Variable Name**. Your variable name can only use uppercase letters, numbers, and underscores.
-3. Enter a **Description** for your variable (optional).
-4. Select **Tags** to associate with your variable (optional).
-5. Enter the **Secret Key** to your variable or upload a QR code image.
-6. Click **+ Generate** to create an OTP. You can copy the generated OTP with the **Copy** icon.
+3. Optionally, enter a **Description** and select **Tags** to associate with your variable. 
+4. Enter the **Secret Key** to your variable or upload a QR code image.
+5. Click **+ Generate** to create an OTP. You can copy the generated OTP with the **Copy** icon.
 
 {{< img src="synthetics/guide/browser-tests-totp/new-variable-totp.png" alt="Create a MFA token" style="width:100%;" >}}
 
 For more information about TOTP-based MFA in a browser test, see [TOTPs For Multi-Factor Authentication (MFA) In Browser Tests][1].
 
 [1]: /synthetics/guide/browser-tests-totp
 {{% /tab %}}
+{{% tab "Virtual Authenticator" %}}
```

Comment:
@ruizb 
Get you please give a quick review for the section below

Reply:
Do we want to keep "virtual authenticator" in the docs, despite using "FIDO2" in the UI?

Reply:
we changed the UI to have Virtual Auth as GV name